src/lib/evil/evil_api.h

```diff
@@ -1,26 +1,44 @@
 #ifndef EVIL_API_H
 #define EVIL_API_H
 
-#ifdef EAPI
-#undef EAPI
+#ifdef EVIL_API
+#error EVIL_API should not be already defined
 #endif
```

Comment:
If we're using `evil_api.h`, there's no reason to check this at all actually, since the include-guards will handle it.